src/polymer/dom-module-scanner.ts

```diff
@@ -87,11 +93,15 @@ export class DomModuleScanner implements HtmlScanner {
 
     await visit((node) => {
       if (isDomModule(node)) {
+        const slots = dom5.queryAll(node, dom5.predicates.hasTagName('slot'))
```

Comment:
Nitpick, but I think it's more correct to find the first `<template>` element that is a direct child of `node` and do this queryAll over that.